shell/vue.config.js

```diff
@@ -1,6 +1,5 @@
 const fs = require('fs');
 const path = require('path');
-const serveStatic = require('serve-static');
```

Comment:
Can this be removed from package.json now?

Reply:
It's still a used in https://github.com/rancher/dashboard/blob/cc4dbdf0c59d19a8eefa7733f487e84f921b5ab4/shell/scripts/serve-pkgs#L17

We might be able to remove it from the root `package.json` though.